src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,275 @@
+          'JS',
+          'None',
+          '2018-05-03'
+        ),
+      ].sort((a, b) => (a.lastupdate > b.lastupdate ? -1 : 1)),
+      page: 0,
+      rowsPerPage: 5,
+    };
+  }
+
+  handleRequestSort = (event, property) => {
+    const orderBy = property;
+    let order = 'desc';
+
+    if (this.state.orderBy === property && this.state.order === 'desc') {
+      order = 'asc';
+    }
+
+    const data =
+      order === 'desc'
+        ? this.state.data.sort((a, b) => (b[orderBy] < a[orderBy] ? -1 : 1))
+        : this.state.data.sort((a, b) => (a[orderBy] < b[orderBy] ? -1 : 1));
+
+    this.setState({ data, order, orderBy });
+  };
+
+  handleChangePage = (event, page) => {
+    this.setState({ page });
+  };
+
+  handleChangeRowsPerPage = event => {
+    this.setState({ rowsPerPage: event.target.value });
+  };
+
+  isSelected = id => this.state.selected.indexOf(id) !== -1;
+
+  render() {
+    const { classes } = this.props;
+    const { data, order, orderBy, rowsPerPage, page } = this.state;
+    const emptyRows =
+      rowsPerPage - Math.min(rowsPerPage, data.length - page * rowsPerPage);
+
+    return (
+      <Paper className={classes.root}>
+        <BugsTableToolbar />
+        <div className={classes.tableWrapper}>
+          <Table className={classes.table} aria-labelledby="tableTitle">
+            <BugsTableHead
+              order={order}
+              orderBy={orderBy}
+              onRequestSort={this.handleRequestSort}
+              rowCount={data.length}
+            />
+            <TableBody>
+              {data
+                .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                .map(n => (
+                  <TableRow hover tabIndex={-1} key={n.id}>
+                    <TableCell component="th" scope="row" padding="none">
+                      {n.project}
+                    </TableCell>
+                    <TableCell>{n.description}</TableCell>
+                    <TableCell>{n.tag}</TableCell>
+                    <TableCell>{n.assignedto}</TableCell>
+                    <TableCell>{n.lastupdate}</TableCell>
+                  </TableRow>
+                ))}
+              {emptyRows > 0 && (
+                <TableRow style={{ height: 49 * emptyRows }}>
+                  <TableCell colSpan={6} />
+                </TableRow>
+              )}
+            </TableBody>
+          </Table>
+        </div>
+        <TablePagination
+          component="div"
+          count={data.length}
+          rowsPerPage={rowsPerPage}
+          page={page}
+          backIconButtonProps={{
+            'aria-label': 'Previous Page',
+          }}
+          nextIconButtonProps={{
+            'aria-label': 'Next Page',
+          }}
+          onChangePage={this.handleChangePage}
+          onChangeRowsPerPage={this.handleChangeRowsPerPage}
+        />
+      </Paper>
+    );
+  }
+}
+
+BugsTable.propTypes = {
```

Comment:
You can have static properties defined inside the class.

```js
class BugsTable extends Component {
  static propTypes = {
    // ...
  }
}
```

Don't forget to also add `defaultProps` for non required properties.